### ISSUE TITLE
Add columns for sessions/talk types to program sheets (docs)

### ISSUE THREAD

**Issue by splattater** [AKSW/leipzig.dataweek.de#227]
- [ ] add columns for tutorial, hackathon, talk, workshop, discussion ... in google docs sheets
- [ ] adjust cmem pipeline
- [ ] show type in program tables

**Comment by white-gecko** [comment]
should this be one "type" column with the values "tutorial, hackathon, talk, workshop, discussion"?

**Comment by splattater** [comment]
@white-gecko Yes, one column should be easier to handle in cmem. Maybe the values can be restricted to the values we need.
